lib/Serializer/XmlEncoder.php

```diff
@@ -202,6 +202,13 @@ private function createParameter($parentEl, $name, $value)
             return $parameterEl;
         }
 
+        if (is_null($value)) {
+            $parameterEl->setAttribute('nillable', 'true');
```

Comment:
Would prefer `nullable`, what do you think?

Reply:
regarding xml uses i will change it to <code>xsi:nil</code>
https://stackoverflow.com/questions/774192/what-is-the-correct-way-to-represent-null-xml-elements